client/src/app/collections/status-collection.coffee

```diff
@@ -0,0 +1,11 @@
+$ = require('jquery')
+Backbone = require('backbone')
+StatusModel = require('../models/status')
+
+module.exports = Backbone.Collection.extend(
+  model: StatusModel
+  comparator: (model) ->
+  	-model.get('id')
+  initialize: ->
+  	@sort(silent: true)
```

Comment:
That was more for the initial sort, I didn't realize defining a comparator handled that part inherently.